website/src/app/kb/user-guides/windows-client/readme.mdx

```diff
@@ -1,34 +1,78 @@
-# Windows Client
+import SupportOptions from "@/components/SupportOptions";
 
-Firezone supports Microsoft Windows with a native client for the x86-64
-architecture.
+# Windows GUI Client
+
+The Windows GUI Client is designed for Windows computesr where a user is present
+to authenticate with your identity provider interactively.
 
 ## Prerequisites
 
-- Windows 10 or higher
-- x86-64 CPU
+- Windows **10** or higher
+- **x86_64** CPU
+- **WebView2** (The installer will install this automatically if needed)
 
 ## Installation
 
-The Windows Client is currently in beta and can be downloaded from the following
-link:
+The Windows Client is currently in beta and can be downloaded from our
+[main repository's releases page](https://github.com/firezone/firezone/releases)
+or using the direct link below:
+
+- [Download the Windows MSI installer for `x86_64`](/dl/firezone-client-gui-windows/latest/x86_64)
 
-- [Download the MSI installer package](/dl/firezone-client-gui-windows/latest/x86_64)
+After downloading, run the MSI to install the Firezone GUI Client.
 
 ## Usage
 
-The Windows Client must be run as an administrator to function correctly. **Be
-sure to click "Allow" when prompted by the User Account Control dialog.**
+### Signing in
+
+1. Run Firezone
+1. At the Welcome screen, click "Sign in". This will open the Firezone sign-in page in your
```

Comment:
I've been trying to use consistent formatting for UI elements mentioned in the docs, like surrounding with <code> blocks. Open to suggestions though.

```suggestion
1. At the Welcome screen, click `Sign in`. This will open the Firezone sign-in page in your
```